noether_msgs/msg/EigenValueEdgeGeneratorConfig.msg

```diff
@@ -28,5 +28,5 @@ int32 max_intersecting_voxels
 # Any two consecutive points with a shortest distance smaller than this value are merged
 float64 merge_dist
 
-# maximum segment length
-float64 max_segment_length
```

Comment:
Not sure how I feel about removing the previous solution - couldn't they have been left in parallel? Now if we want to go back for any reason (or combine the approaches), we have to edit Noether again.

Reply:
I don't entirely disagree, but the call to remove it came from on high

Reply:
Gotcha.  No shame in acquiescing to on high.